tests/ui/drop/drop_order.rs

```diff
@@ -1,6 +1,11 @@
 //@ run-pass
 //@ compile-flags: -Z validate-mir
+//@ revisions: edition2021 edition2024
+//@ [edition2021] edition: 2021
+//@ [edition2024] compile-flags: -Z unstable-options --cfg edition2024
```

Comment:
Suggestion [REV3,ED2024]: this `--cfg edition2024` is redundant, the `edition2024` being declared in revisions will cause compiletest to automatically set a `cfg` of the same name.

Reply:
Yanked